Formula/python-rpds-py.rb

```diff
@@ -0,0 +1,33 @@
+class PythonRpdsPy < Formula
+  desc "Python bindings to Rust's persistent data structures"
+  homepage "https://github.com/crate-py/rpds"
+  url "https://files.pythonhosted.org/packages/da/3c/fa2701bfc5d67f4a23f1f0f4347284c51801e9dbc24f916231c2446647df/rpds_py-0.9.2.tar.gz"
+  sha256 "8d70e8f14900f2657c249ea4def963bed86a29b81f81f5b76b5a9215680de945"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "python@3.11"
+
+  def python
+    "python3.11"
+  end
+
+  def install
+    system python, "-m", "pip", "install", "--prefix=#{prefix}", "--no-deps", "."
```

Comment:
This fetches `maturin` pre-built wheel as build dependency.

Reply:
e.g. if you run with `-v`, 
```
  Collecting maturin<2.0,>=1.0
    Downloading maturin-1.1.0-py3-none-macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2.whl (14.5 MB)
       ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 14.5/14.5 MB 35.2 MB/s eta 0:00:00
  Installing collected packages: maturin
```

Reply:
good point, I will check it out later. 👍

Reply:
fyi, there's now a `std_pip_args`we can use